src/Game.cpp

```diff
@@ -39,17 +39,179 @@ static void sub_53A890(const char *args) {
     sub_612A80(loc_6C3690);
 }
 
+// mc2: 0x005EDA50
+void game_set_window_title(char* title) {
+    HWND hWnd = global_hWnd;
+    global_WindowText = title;
+    if (hWnd != NULL)
+        SetWindowTextA(global_hWnd, title);
+}
+
+// mc2: 0x00539FE0 and 0x00539D90
+void check_sku_version() {
+    global_SKUVersion = 0; //SKU VERSION SLUS-20209
+
+    char* skuVersion = NULL;
+
+    switch (global_SKUVersion) {
+    case 0:
+        skuVersion = "SLUS-20209";
+        break;
+    case 1:
+        skuVersion = "SLES-51054";
+        break;
+    case 2:
+        skuVersion = "SLPM-20209";
+        break;
+    default:
+        // SKU unrecognised
+        mc2_log_error("Unhandled sku");
+        return;
+    }
+
+    mc2_log_C(skuVersion);
```

Comment:
According to `sub_539FE0`, this should be `mc2_loc_C("mcConfig:AutoDetectSku - product code '%s'");`